### PR TITLE
Make validation markup more generic

### PR DESCRIPTION
**Pull request by katzmo**

### CODE REVIEW

File: package/src/method-element.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ class MethodElement {
   validate (submitter) {
     $('.mo-dialog-wrapper').addClass('visible')
     if (typeof Drupal.clientsideValidation !== 'undefined') {
-      $('#clientsidevalidation-' + this.form_id + '-errors ul').empty()
```
Comment (torotil) [inline package/src/method-element.js:156]: Is there a chance that clientside validation is enabled for *other* forms on the same page, but isn’t enabled for *this* form?

File: package/src/method-element.js [hunk 2]
```diff
@@ -215,14 +216,30 @@ class MethodElement {
...
+        }
       }
     }
-    // No clientside validation or error is not related to a payment field.
-    if ($('#messages').length === 0) {
-      $('<div id="messages"><div class="section clearfix"></div></div>').insertAfter('#header')
+    // Without clientside validation render a message above the fieldset.
+    else {
+      const $message = $('<div class="messages error">').text(error.message)
+      $message.addClass('stripe-error').insertBefore(this.$element.closest('form'))
```
Comment (torotil) [inline package/src/method-element.js:262]: This is above the form (as stated in the commit message) not the fieldset (as stated in the comment), right?

**Review by torotil — CHANGES_REQUESTED** [review]

File: package/src/method-element.js (hunk 2 above)
Comment (katzmo) [reply package/src/method-element.js:262]: yes ;)

File: package/src/method-element.js (hunk 1 above)
Comment (katzmo) [reply package/src/method-element.js:156]: Theoretically, I guess it's possible…?

**Review by torotil — APPROVED** [review]

**Pull request merged**